src/test/java/uk/gov/hmcts/reform/pbis/ApplicationTest.java

```diff
@@ -7,7 +7,7 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
-public class ApplicationTests {
+public class ApplicationTest {
```

Comment:
I can't see where these config values are being used?

Reply:
I've split the changes so that they could be merged in relatively small chunks. This one is the implementation of [RPE-72](https://tools.hmcts.net/jira/browse/RPE-72). As it has no dependencies, this is the one that should go before other stories.